datafusion/expr/src/type_coercion/aggregates.rs

```diff
@@ -395,6 +440,17 @@ pub fn avg_sum_type(arg_type: &DataType) -> Result<DataType> {
     }
 }
 
+pub fn is_bit_and_or_xor_support_arg_type(arg_type: &DataType) -> bool {
+    matches!(
+        arg_type,
+        arg_type if NUMERICS.contains(arg_type)
+    )
```

Comment:
Is there any reason to use a `matches!` here? it seems like this could just be `NUMERICS.contains(arg_type)` 🤔